### PR TITLE
Noahp/machinations encounter

### PR DESCRIPTION
**Pull request by NoahPeres**
Every card has been typed and separately proof-read at least one additional time - hopefully the formatting and stuff is in line with what is there so far generally speaking

TODO 
- (l. 176, 347, 514) I 'hacked' the macro on my local arkhamdb to get it to render, but sort out something better for the shroud per investigator on the arkhamdb side

### CODE REVIEW

File: pack/side/mtt_encounter.json [hunk 1]
```diff
@@ -1,4 +1,258 @@
...
+        "pack_code": "mtt",
+        "position": 4,
+        "quantity": 1,
+        "stage": 1,
+        "text": "Abilities on [[Scientist]] assets can be triggered by any investigator at that asset's location.\n[reaction] When you move to a location, if one or more [[Scientist]] assets are at your location: Move any number of them with you.\n<b>Objective</b> - Restore the timeline by completing the objective on each story card. If Thomas Corrigan and Mary Zielinski are at non-Tindalos locations in all 3 eras, and each era has no story cards in play, advance.",
+        "type_code": "act"
+    },
+    {
+        "clues": 0,
+        "code": "87005a",
```
Comment (fspoettel) [inline pack/side/mtt_encounter.json:78]: Some cases where there's an `a` and `b` printed on the card reflect this in the id (`87005a`, `87036a`, `87037a`), most don't (`87006`, `87015`, `87024`, `87033`, `87034`, `87035`). Is this intentional?

File: pack/side/mtt_encounter.json [hunk 2]
```diff
@@ -63,5 +604,476 @@
...
+        "traits": "Machination.",
+        "type_code": "story",
+        "victory": 1
+    },
+    {
+        "back_text": "[action] [action]: Ready and move Edwin Bennet to your location <i>(even if he is in another era)</i>.\n[action] If Edwin Bennet is at your location: <b>Parley.</b> Test [willpower] (3) or [intellect] (3). If you succeed, spend X clues, as a group where X is the global number of players. Then, place 1 resource on him, as a redemption.\n<b>Objective</b> - If Edwin Bennet has 3 or more redemption on him and he is at any Miskatonic University with Thomas Corrigan and Mary Zielinski, he is redeemed. Remove all redemption from him and flip him over. Add all copies of this story card in each era to the victory display.",
+        "back_traits": "Machination.",
```
Comment (drawntotheflame) [inline pack/side/mtt_encounter.json:628]: You can cut this line.

File: pack/side/mtt_encounter.json [hunk 3]
```diff
@@ -63,5 +604,476 @@
...
+        "traits": "Machination.",
+        "type_code": "story",
+        "victory": 1
+    },
+    {
+        "back_text": "[action] If Edwin Bennet is at your location and he is ready: <b>Parley.</b> Place 1 of your clues on him. You may test [intellect] (4). If you succeed, place 1 more of your clues on him. If you fail, exhaust him.\n[action]: Ready and move Edwin Bennet to your loaction <i>(even if he is in another era)</i>.\n<b>Forced</b> - At the end of the round: Remove 1 clue from Edwin Bennet (2 clues instead if you are playing in <i>Epic Multiplayer Mode</i>). If you cannot, place 1 doom on the current agenda. This effect can cause the current agenda to advance.\n<b>Forced</b> - When Edwin Bennet would be abducted: Remove all damage and horror from him and place 2 doom on the current agenda instead.\n<b>Objective</b> - If \"Thomas and Mary have won a Nobel Prize\" and there is no [[Plot]] story card in play in any era, add this story card to the victory display.",
+        "back_traits": "Machination.",
```
Comment (drawntotheflame) [inline pack/side/mtt_encounter.json:646]: You can cut this line.

File: pack/side/mtt_encounter.json [hunk 4]
```diff
@@ -63,5 +604,476 @@
...
+        "traits": "Machination.",
+        "type_code": "story",
+        "victory": 1
+    },
+    {
+        "back_text": "[action] If Edwin Bennet is at your location and he is ready: <b>Parley.</b> Place 1 of your clues on him. You may test [intellect] (4). If you succeed, place 1 more of your clues on him. If you fail, exhaust him.\n[action]: Ready and move Edwin Bennet to your loaction <i>(even if he is in another era)</i>.\n<b>Forced</b> - At the end of the round: Remove 1 clue from Edwin Bennet (2 clues instead if you are playing in <i>Epic Multiplayer Mode</i>). If you cannot, place 1 doom on the current agenda. This effect can cause the current agenda to advance.\n<b>Forced</b> - When Edwin Bennet would be abducted: Remove all damage and horror from him and place 2 doom on the current agenda instead.\n<b>Objective</b> - If \"Thomas and Mary have won a Nobel Prize\" and there is no [[Plot]] story card in play in any era, add this story card to the victory display.",
```
Comment (drawntotheflame) [inline pack/side/mtt_encounter.json:645]:
```suggestion
        "back_text": "[action] If Edwin Bennet is at your location and he is ready: <b>Parley.</b> Place 1 of your clues on him. You may test [intellect] (4). If you succeed, place 1 more of your clues on him. If you fail, exhaust him.\n[action]: Ready and move Edwin Bennet to your location <i>(even if he is in another era)</i>.\n<b>Forced</b> - At the end of the round: Remove 1 clue from Edwin Bennet (2 clues instead if you are playing in <i>Epic Multiplayer Mode</i>). If you cannot, place 1 doom on the current agenda. This effect can cause the current agenda to advance.\n<b>Forced</b> - When Edwin Bennet would be abducted: Remove all damage and horror from him and place 2 doom on the current agenda instead.\n<b>Objective</b> - If \"Thomas and Mary have won a Nobel Prize\" and there is no [[Plot]] story card in play in any era, add this story card to the victory display.",
```

File: pack/side/mtt_encounter.json [hunk 5]
```diff
@@ -63,5 +604,476 @@
...
+        "double_sided": true,
+        "encounter_code": "machinations_through_time",
+        "encounter_position": 34,
+        "faction_code": "mythos",
+        "flavor": "Nothing in the past appears to be out of the ordinary. You wonder if Edwin Bennet is trustworthy. Surely his vast knowledge could prove extremely useful in aiding you. Reluctantly, you convince yourself the payoff is worth the risk, but it will take a great deal of persuasion to keep Edwin from betraying you.",
+        "name": "Uneasy Alliance",
+        "pack_code": "mtt",
+        "position": 35,
+        "quantity": 1,
+        "text": "<b>Setup</b>: Follow the instructions for your relevant era(s), below. Then, flip this story card over.<hr><b>Past</b>: Place Thomas Corrigan at Childhood Home. Place Mary Zielinski at O'Malley's Watch Shop. Place the Edwin Bennet asset at Arkham Gazette with 12 clues on him, then remove 3 [per_investigator] clues from him.<hr><b>Present and Future</b>: Set Thomas Corrigan and Mary Zielinsky aside, out of play: they have been abducted. If you are playing in <i>Epic Multiplayer Mode</i>, set the Edwin Bennet asset aside, out of play.",
```
Comment (drawntotheflame) [inline pack/side/mtt_encounter.json:657]:
```suggestion
        "text": "<b>Setup</b>: Follow the instructions for your relevant era(s), below. Then, flip this story card over.<hr><b>Past</b>: Place Thomas Corrigan at Childhood Home. Place Mary Zielinski at O'Malley's Watch Shop. Place the Edwin Bennet asset at Arkham Gazette with 12 clues on him, then remove 3 [per_investigator] clues from him.<hr><b>Present and Future</b>: Set Thomas Corrigan and Mary Zielinski aside, out of play: they have been abducted. If you are playing in <i>Epic Multiplayer Mode</i>, set the Edwin Bennet asset aside, out of play.",
```

File: pack/side/mtt_encounter.json [hunk 6]
```diff
@@ -63,5 +604,476 @@
...
+    },
+    {
+        "code": "87036b",
+        "cost": null,
+        "encounter_code": "machinations_through_time_single_group",
+        "encounter_position": 2,
+        "faction_code": "neutral",
+        "flavor": "A truly brilliant, yet unstable, mastermind.",
+        "health": 2,
+        "illustrator": "Drazenka Kimpel",
```
Comment (drawntotheflame) [inline pack/side/mtt_encounter.json:692]: Please add `"hidden": true,`, which we add for all 'back' cards when they are split over two entries.

File: pack/side/mtt_encounter.json [hunk 7]
```diff
@@ -63,5 +604,476 @@
...
+        "type_code": "enemy"
+    },
+    {
+        "code": "87037b",
+        "cost": null,
+        "encounter_code": "machinations_through_time_epic_multiplayer",
+        "encounter_position": 2,
+        "faction_code": "neutral",
+        "health": 2,
+        "illustrator": "Drazenka Kimpel",
```
Comment (drawntotheflame) [inline pack/side/mtt_encounter.json:733]: Please add `"hidden": true,`

File: pack/side/mtt_encounter.json [hunk 8]
```diff
@@ -63,5 +604,476 @@
...
+        "quantity": 1,
+        "sanity": 2,
+        "subname": "Esteemed Colleague",
+        "text": "<b>Forced</b> - After Edwin Bennet exhausts: Choose an investigator to draw the top card of the encounter deck.\n[reaction] When you draw an encounter card, exhaust Edwin Bennet: Cancel all of that card's effects and discard it.",
+        "traits": "Scientist. Ally.",
+        "type_code": "asset"
+    },
+    {
+        "back_text": "Abilities on locations with an anomaly cannot be triggered.\n[action] Spend any number of clues: Test [willpower] (3) or [agility] (3). You get +3 skill value to this test for each clue spent this way. If you succeed, remove 1 anomaly from your location (2 anomalies instead if you succeed by 3 or more). Then, if you succeed by 4 or more and \"Corrigan Industries has been founded,\" remove 1 anomaly from a location with the same title as your location, in any era.\n<b>Objective</b> - If it is agenda 2a and each location has no anomalies, add this story card to the victory display.",
+        "back_traits": "Plot.",
```
Comment (drawntotheflame) [inline pack/side/mtt_encounter.json:747]: You can cut this line.

File: pack/side/mtt_encounter.json [hunk 9]
```diff
@@ -63,5 +604,476 @@
...
+        "position": 38,
+        "quantity": 1,
+        "text": "<b>Setup</b>: Place 1 [per_investigator] horror on each Miskatonic University location, as anomalies. If you are playing in <i>Epic Multiplayer Mode</i>, also place 1 [per_investigator] horror on the following locations, as anomalies: Arkham Gazette, Arkham Advertiser, O'Malley's Watch Shop, and Tick-Tock Club.\nRemove the following enemies from the game: Tyr'thrha, Old Sadie Sheldon, and all 3 copies of Sheldon Gang. Flip this story card over.",
+        "traits": "Plot.",
+        "type_code": "story",
+        "victory": 1
+    },
+    {
+        "back_text": "[action] If \"the debt has been paid,\" spend 1 [per_investigator] clues, as a group: Search the play area, encounter deck, or encounter discard pile for a Sheldon Gang enemy and add it to the victory display. If you searched the encounter deck, shuffle it. This action does not cause attacks of opportunity.\n<b>Objective</b> - If all 3 Sheldon Gang enemies are in the victory display, add this story card to the victory display.",
+        "back_traits": "Plot.",
```
Comment (drawntotheflame) [inline pack/side/mtt_encounter.json:765]: You can cut this line.

File: pack/side/mtt_encounter.json [hunk 10]
```diff
@@ -63,5 +604,476 @@
...
+        "faction_code": "mythos",
+        "health": 1,
+        "illustrator": "Rafal Hrynkiewicz",
+        "is_unique": true,
+        "name": "Old Sadie Sheldon",
+        "pack_code": "mtt",
+        "position": 40,
+        "quantity": 1,
+        "subname": "Bootlegging Kingpin",
+        "text": "Aloof.\nEach Sheldon Gang enemy gets +1 fight and +1 evade.\n[action]: <b>Parley</b>. Test [intellect] (2). If you succeed, spend 3 [per_investigator] resources, as a group, to add Old Sadie Sheldon to the victory display and announce \"the debt has been paid.\"",
```
Comment (drawntotheflame) [inline pack/side/mtt_encounter.json:797]:
```suggestion
        "text": "Aloof.\nEach Sheldon Gang enemy gets +1 fight and +1 evade.\n[action]: <b>Parley.</b> Test [intellect] (2). If you succeed, spend 3 [per_investigator] resources, as a group, to add Old Sadie Sheldon to the victory display and announce \"the debt has been paid.\"",
```

File: pack/side/mtt_encounter.json [hunk 11]
```diff
@@ -63,5 +604,476 @@
...
+        "position": 41,
+        "quantity": 3,
+        "text": "Surge. Hunter. Massive.\nSheldon Gang gets +2 [per_investigator] health.\n<b>Forced</b> - After an investigator at Sheldon Gang's location plays a card: Ready Sheldon Gang and discard the top 3 cards of the encounter deck. Put each [[Criminal]] enemy discarded this way into play at that location.",
+        "traits": "Humanoid. Criminal. Syndicate. Elite.",
+        "type_code": "enemy",
+        "victory": 0
+    },
+    {
+        "back_text": "[action] If Tyr'thrha is at your location, spend 1 clue: Test [intellect] (3). If you succeed, deal 2 damage to Tyr'thrha (3 damage instead if you succeed by 3 or more). This action does not provoke attacks of opportunity.\n<b>Objective</b> - If Tyr'thrha is defeated, add this story card to the victory display.",
+        "back_traits": "Plot.",
```
Comment (drawntotheflame) [inline pack/side/mtt_encounter.json:823]: You can cut this line.

File: pack/side/mtt_encounter.json [hunk 12]
```diff
@@ -63,5 +604,476 @@
...
+        "code": "87042",
+        "double_sided": true,
+        "encounter_code": "machinations_through_time",
+        "encounter_position": 41,
+        "faction_code": "mythos",
+        "flavor": "The extreme warping of spacetime in Tindalos has affected everything within. The satyrs and Hounds of Tindalos aren't the only beings to stalk about the realm of angular time - a being exponentially larger and more powerful twists spacetime itself.\nThose who continually walk the paths of this vast maze are bound to draw the ire of those who call it home, andnothing compares to the wrath of the Arch-Lord of Tindalos.",
```
Comment (drawntotheflame) [inline pack/side/mtt_encounter.json:829]:
```suggestion
        "flavor": "The extreme warping of spacetime in Tindalos has affected everything within. The satyrs and Hounds of Tindalos aren't the only beings to stalk about the realm of angular time - a being exponentially larger and more powerful twists spacetime itself.\nThose who continually walk the paths of this vast maze are bound to draw the ire of those who call it home, and nothing compares to the wrath of the Arch-Lord of Tindalos.",
```

File: pack/side/mtt_encounter.json [hunk 13]
```diff
@@ -63,5 +604,476 @@
...
+        "code": "87043",
+        "encounter_code": "machinations_through_time",
+        "encounter_position": 42,
+        "enemy_damage": 1,
+        "enemy_evade": 3,
+        "enemy_fight": 3,
+        "enemy_horror": 2,
+        "faction_code": "mythos",
+        "health": -2,
+        "illustrator": "Helge C Baker",
```
Comment (drawntotheflame) [inline pack/side/mtt_encounter.json:849]:
```suggestion
        "illustrator": "Helge C Balzer",
```

**Review by drawntotheflame — COMMENTED** [review]
Great job, just a few little tweaks.

**Comment by NoahPeres** [comment]
Wow Ty for awesome feedback will go through these hopefully within the next couple weeks some time.

**Pull request merged**